http_rules/test_http.py

```diff
@@ -25,12 +25,12 @@ class HttpRules(functional.FunctionalTest):
         '  host == "static.*" -> host_p;\n'
         '  host == "*tempesta-tech.com" -> host_s;\n'
         '  host == "foo.example.com" -> host_e;\n'
-        '  hdr_host == "bar.*" -> hdr_h_p;\n'
-        '  hdr_host == "buzz.natsys-lab.com" -> hdr_h_e;\n'
-        '  hdr_host == "*natsys-lab.com" -> hdr_h_s;\n'
-        '  hdr_ref ==  "example.com" -> hdr_r_e;\n'
-        '  hdr_ref ==  "*.com" -> hdr_r_s;\n'
-        '  hdr_ref ==  "http://example.com*" -> hdr_r_p;\n'
+        '  hdr Host == "bar.*" -> hdr_h_p;\n'
+        '  hdr host == "buzz.natsys-lab.com" -> hdr_h_e;\n'
+        '  hdr Host == "*natsys-lab.com" -> hdr_h_s;\n'
+        '  hdr Referer ==  "example.com" -> hdr_r_e;\n'
+        '  hdr Referer ==  "*.com" -> hdr_r_s;\n'
+        '  hdr referer ==  "http://example.com*" -> hdr_r_p;\n'
```

Comment:
Need to add rule for raw header comparison. That branch of code is not tested here or anywhere eslse if i'm not mstaken

Reply:
Done.